maven-setup/action.yml

```diff
@@ -56,6 +56,20 @@ inputs:
   develocity-allow-untrusted-server:
     description: 'Whether to allow communicating with untrusted server'
     required: false
+  develocity-enforce-url:
+    description: 'Whether to enforce the URL over any defined locally to the project'
```

Comment:
is the description message correct?

Reply:
Yes, it's a boolean flag which forces to use configured URL in case there is already an extension applied, but it's configured URL is different